src/main/java/org/jenkinsci/test/acceptance/po/Build.java

```diff
@@ -157,6 +157,14 @@ public String getConsole() {
         return visit(getConsoleTextUrl()).getPageSource();
     }
 
+    public URL getGitBuildDataUrl() {
```

Comment:
Oh, I didn't take into account that GitBuildData is only available if Git-Plugin is installed.. I'll move it somewhere else